### ISSUE TITLE
Openhabian creates .bash_profile that does not include .profile

### ISSUE THREAD

**Issue by MaxBlasto** [openhab/openhabian#1334]
## Issue information:
openhabian creates / overwrites a .bash_profile which does not load .profile or does not take earlier changes into account.
Including .profile needs to be added manually to .bash_profile.
This can cause issues when other software than openhab is running on the machine.

## System information:
cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian

uname -m
armv7l

**Comment by ecdye** [comment]
Yes, I am awaye of the issue. It will be fixed soon with the correction of dynamic openHAB logo switching. See #1273

**Comment by mstormi** [comment]
@MaxBlasto we do not support running stuff other than the included on openhabian boxes
You can do at your own risk but for sure you must not use the openhabian default user for that.
So strictly speaking we should reject your request. Then again it's simple so I don't mind adding it.

> It will be fixed soon with the correction of dynamic openHAB logo switching.

@ecdye u sure? includes/bash_profile has not been containing any sourcing of ~/.profile and #1338 does not change anything about that so add a simple `if [[ -f $HOME/.profile ]]; then source $HOME/.profile; fi` to the template as part of #1338

**Comment by ecdye** [comment]
> @MaxBlasto we do not support running stuff other than the included on openhabian boxes
> You can do at your own risk but for sure you must not use the openhabian default user for that.
> So strictly speaking we should reject your request. Then again it's simple so I don't mind adding it.
> 
> > It will be fixed soon with the correction of dynamic openHAB logo switching.
> 
> @ecdye u sure? includes/bash_profile has not been containing any sourcing of ~/.profile and #1338 does not change anything about that so add a simple `if [[ -f $HOME/.profile ]]; then source $HOME/.profile; fi` to the template as part of #1338

Yes if he was manually adding it is would be overwritten every time the `openhabian-config` tool was updated or the branch was switched so it would make it no longer work for him.

**Comment by MaxBlasto** [comment]
Thanks for considering this. 
You're of course right, I shouldn't be using the openhabian user for other stuff.

Only as an explanation why I came to mark this as a problem:
I stumbled upon this issue because I was running OH2 on an existing system where I installed it using openhabian-config without using a fresh install of openhabian itself, which worked fine until openhabian-config updated itself to the version that supports OH3 and suddenly my settings from .profile were gone. It took me a while to find out what had happened.

Thanks again for helping me out with this. I definitely appreciate it.
